src/screens/Home/Home.tsx

```diff
@@ -38,6 +39,50 @@ interface HomeProps {
 function Home(props: HomeProps) {
   const { classes, lastUpdated, isOnline, currentSite } = props;
 
+  const calculateCustomerData = () => {
+    // customers to charge:  haven't charged yet/ no meterreading
+    // outstanding payments: done the meter reading / charged, haven't paid yet
+    let toCharge = 0;
+    let outstandingPayments = 0;
+    let customers = currentSite.customers;
+    for (let i = 0; i < customers.length; i++) {
+      const currCustomer = customers[i]
+      //depends if the meter readings list is sorted with earliest => latest
+      let latestMeterReading = currCustomer.meterReadings[currCustomer.meterReadings.length - 1];
+      if (latestMeterReading != null) {
+        let lessMonth = lessThanCurrentMonth(latestMeterReading.date)
+        if (lessMonth) {
+          toCharge += 1;
+        }
+        if (checkPayment(currCustomer)) {
+          outstandingPayments += 1;
+        }
+      } else {
+        toCharge += 1;
+      }
+    }
+    let total = outstandingPayments + toCharge;
+    return { 'outstandingPayments': outstandingPayments, 'toCharge': toCharge, 'totalAmount': total }
+  }
+
+  //true has outstadining false has paid
+  const checkPayment = (customer: CustomerRecord) => {
+    return parseInt(customer.outstandingBalance) > 0;
+  }
+
+  const calculateUnpaidReports = () => {
```

Comment:
We should ask Ashely, but I'm pretty sure unpaid reports in this context are reports that the user hasn't paid off, not that there are customers left to collect from.

The user needs to have gone through the below payment screens to have their report considered "paid" i think. 

We could possibly add a `Paid` field to see if they've paid or we could check for a copy of the bankslip. I don't know what to do for part-payments, though. I'll ask the NPO tomorrow if that's a concern.

![image](https://user-images.githubusercontent.com/35501399/109895930-f4b42880-7c44-11eb-8ec2-58f576c6b57e.png)